Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -275,6 +294,61 @@ def _update_stats_model(self):
                 row.append(item)
             model.appendRow(row)
 
+    def _update_class_selection(self):
+        if is_discrete(self.train_data.domain.class_var):
+            self.cbox.setVisible(True)
+            values = self.train_data.domain.class_var.values
+            self.class_selection_combo.clear()
+            self.class_selection_combo.addItem("(None)")
+            self.class_selection_combo.addItems(values)
+
+            class_index = 0
+            if self.class_selection != '(None)' and self.class_selection != 0:
+                class_map = {val: i for i, val in enumerate(self.train_data.domain.class_var.values)}
+                class_index = class_map[self.class_selection]+1
```

Comment:
Use self.train_data.domain.class_var.values.index(self.class_selection) instead of creating a dictionary